ci/doc.yml

```diff
@@ -0,0 +1,28 @@
+name: xarray-docs
```

Comment:
It's not clear to me that we actually need a new doc env specification. We also have the old https://github.com/pvlib/pvlib-python/blob/master/docs/environment.yml from when we did use conda on rtd. Could probably be deleted.

Reply:
Yes, I tried this initially but it got left in after I transitioned to a pip install only. I'll delete it.